mev_inspect/transfers.py

```diff
@@ -18,9 +25,21 @@ def get_eth_transfers(traces: List[ClassifiedTrace]) -> List[EthTransfer]:
 def get_transfers(traces: List[ClassifiedTrace]) -> List[ERC20Transfer]:
     transfers = []
 
+    specs_by_abi_name_and_protocol: Dict[
+        Tuple[str, Optional[Protocol]], ClassifierSpec
+    ] = {(spec.abi_name, spec.protocol): spec for spec in ALL_CLASSIFIER_SPECS}
+
     for trace in traces:
-        if trace.classification == Classification.transfer:
-            transfers.append(ERC20Transfer.from_trace(trace))
+        if not isinstance(trace, DecodedCallTrace):
+            continue
+
+        abi_name_and_protocol = (trace.abi_name, trace.protocol)
+        spec = specs_by_abi_name_and_protocol.get(abi_name_and_protocol)
```

Comment:
this logic is a little gross and will be moved to a shared function as swap and liquidate are added